e2e/host/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+type data struct {
+	host string
+	path string
+}
+
+type test struct {
+	name     string
+	args     data
+	expected string
+	referer  bool
+}
+
+var tests = []test{
+	{
+		name: "Redirect to a host record's to= field",
+		args: data{
+			host: "to.host.host.example.com",
+			path: "/",
+		},
+		expected: "https://to-redirect.host.host.example.com",
+	},
+	{
+		name: "Redirect to a host record's to= field without specified code=",
+		args: data{
+			host: "nocode.host.host.example.com",
+			path: "/",
+		},
+		expected: "https://nocode.host.host.example.com",
+	},
+	{
+		name: "Redirect to a host record's to= field without specified v=",
+		args: data{
+			host: "noversion.host.host.example.com",
+			path: "/",
+		},
+		expected: "https://noversion.host.host.example.com",
+	},
+	{
+		name: "Redirect to a host record's unspecified to=",
+		args: data{
+			host: "noto.host.host.example.com",
+			path: "/",
+		},
+		expected: "http://noto.host.host.example.com/",
```

Comment:
As I mentioned in my first comment, we should reconsider this test (it was in our old e2e tests) and return an error and trigger the fallback when `to=` is empty.

Reply:
Shouldn't this be 404 in general with no other fallback configured?

Reply:
?

Reply:
https://github.com/txtdirect/txtdirect/blob/97a881d2486ed2676768b9bb6c7845e1ec138cc5/record.go#L192
I thought this would take care of it but it only gets called when `to=` exist. Will add a condition to trigger fallback when the type is `host` and `to=` doesn't exist.

Reply:
+1